doc/ref/states/all/salt.states.boto3_efs.py

```diff
@@ -0,0 +1,6 @@
+salt.states.boto3_efs module
+============================
+
+.. automodule:: salt.states.boto3_efs
+    :members:
+    :undoc-members:
```

Comment:
you will need to add boto3_efs to doc/ref/states/all/index.rst for this to be rendered.

Reply:
Done